### PR TITLE
Added info box check for series completing vaccines over 14 days (EXPOSUREAPP-10538)

### PR DESCRIPTION
**Pull request by SamuraiKek**
In cases where a new series completing vaccination is scanne and there's an existing series completing vaccination > 14 days, the vaccination info box should say that the vaccination protection is complete and no days counter is shown.

How to test:

1. Scan a J&J certificate > 14 days old:
`cwa vc gen --pii 42 -s v.0.dn=1 v.0.sd=1 v.0.mp='EU/1/20/1525' v.0.dt='2021-10-04' -e wru` -> info box says protection is complete, no days counter is shown.
2. Scan a Pfizer certificate < 14 days old: 
`cwa vc gen --pii 42 -s v.0.dn=2 v.0.sd=2 v.0.mp='EU/1/20/1528' v.0.dt='2021-11-04' -e wru` -> info box still says protection is complete, no days counter is shown.
3. Delete both certificates.
4. Scan a J&J certificate < 14 days old:
`cwa vc gen --pii 42 -s v.0.dn=1 v.0.sd=1 v.0.mp='EU/1/20/1525' v.0.dt='2021-11-04' -e wru` -> info box says you've received all planned vaccinations but the days counter is shown.
5. Scan a Pfizer certificate > 14 days old:
`cwa vc gen --pii 42 -s v.0.dn=2 v.0.sd=2 v.0.mp='EU/1/20/1528' v.0.dt='2021-10-04' -e wru` -> info box says protection is complete, no days counter is shown.

### CODE REVIEW

**Review by AlexanderAlferov — DISMISSED** [review]
LGTM 🚀

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt [hunk 1]
```diff
@@ -83,12 +84,25 @@ data class VaccinatedPerson(
...
     private fun getNewestDoseVaccinatedOn(): LocalDate =
         vaccinationCertificates.maxOf { it.vaccinatedOn }
 
+    private fun isSeriesCompletingOverTwoWeeks(nowUTC: Instant = Instant.now()): Boolean {
+        val certificate = vaccinationCertificates
+            .filter { it.isSeriesCompletingShot }
+            .firstOrNull {
+                Days.daysBetween(
+                    it.rawCertificate.vaccination.vaccinatedOn,
+                    nowUTC.toLocalDateUtc()
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt:100]: Can you move that into the fun -> eg. (today: LocalDate) so it is recalculated for every certificate?

Comment (chiljamgossow) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt:100]: isSeriesCompletingShotOverTwoWeeksOld(today: LocalDate) ?

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt:100]: Yes

**Review by AlexanderAlferov — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]
Works

**Pull request merged**